tests/test_binary_pointwise_ops.py

```diff
@@ -507,6 +507,38 @@ def test_accuracy_pow(shape, dtype):
     gems_assert_close(res_out, ref_out, dtype, equal_nan=True)
 
 
+@pytest.mark.parametrize("shape", POINTWISE_SHAPES)
+@pytest.mark.parametrize("dtype", FLOAT_DTYPES)
+def test_accuracy_maximum(shape, dtype):
+    if dtype == torch.bfloat16:
+        return
+    inp1 = torch.randn(shape, dtype=dtype, device="cuda")
+    inp2 = torch.randn(shape, dtype=dtype, device="cuda")
+    ref_inp1 = to_reference(inp1, True)
+    ref_inp2 = to_reference(inp2, True)
```

Comment:
it's not necessary to cast reference input tensors to fp64 here.

Reply:
got it